Containerfile

```diff
@@ -1,18 +1,31 @@
-## Run
-####################
-# Obtained from: https://catalog.redhat.com/software/containers/ubi9/nodejs-18/62e8e7ed22d1d3c2dfe2ca01?container-tabs=gti
-FROM registry.access.redhat.com/ubi9/nodejs-18:9.5-1733148761
+ARG NODE_VERSION=18
+ARG NODE_IMAGE_TAG=9.5-1733148761
 
-# Labels
-LABEL VERSION="v0.2"
-LABEL description="ClusterIQ Web Console"
+ARG NGINX_VERSION=124
```

Comment:
Quay.io doesn't support ARGS, but as long as we don't need to setup any specific value during the building, it should work with the default values.